src/solver/include/toolkit.h

```diff
@@ -116,15 +132,15 @@ int DLLEXPORT swmm_getObjectIndex(SM_ObjectType type, char *id, int *index);
  id must be pre-allocated by the caller.
  @return Error code
 */
-int DLLEXPORT swmm_getNodeType(SM_NodeType index, int *Ntype);
+int DLLEXPORT swmm_getNodeType(int index, SM_NodeType *Ntype);
```

Comment:
@cbuahin I think what happened is that the arguments of the function got transposed when it got refactored. Since this is a "getter" *Ntype is a pointer to a return value. It's an informal convention, but typically when pointers are return values they the the last argument.